src/controllers/expenses.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const expensesServices = require('../services/expenses');
+
+const getAll = (req, res) => {
+  const expenses = expensesServices.filterExpenses(req.query);
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+  const foundExpenses = expensesServices.getById(id);
+
+  if (!foundExpenses) {
+    res.status(404).send('Expenses not found');
+
+    return;
+  }
+
+  res.send(foundExpenses);
+};
+
+const add = (req, res) => {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  const isRequiredField = !spentAt
```

Comment:
You can simplify your condition check for required fields using an array and the every method, e.g. 'const isRequiredField = [spentAt, title, amount, category, note].every(field => !field);'

Reply:
That is a fantastic solution to this problem, Thank you for the idea.
Done.